QUARKUS-1407.md

```diff
@@ -0,0 +1,38 @@
+# Quarkus-1407 - Testing and verifying Oracle SQL client
```

Comment:
We already got Oracle SQL client, this is about reactive client for Vertx, right?

Also upstream docs for 2.7 say this is _experimental_, and Jira says _tech preview_. This needs to be cleared up too.

Reply:
Could you send me the link? Code site says[1] it's preview and announcement[2] doesn't mention current status at all

[1] https://code.quarkus.io/?extension-search=oracle
[2]https://quarkus.io/blog/quarkus-2-7-0-final-released/#oracle-reactive-sql-client

Reply:
https://issues.redhat.com/browse/QUARKUS-1164 - this is the previous Oracle JDBC support

https://quarkus.io/guides/reactive-sql-clients the docs

Reply:
I asked a question in the issue, but probably reactive guide is not yet updated